checkstyle.xml

```diff
@@ -27,14 +27,14 @@
 
         <!-- Checks for Naming Conventions.                  -->
         <!-- See http://checkstyle.sf.net/config_naming.html -->
-        <module name="ConstantName"/>
-        <module name="LocalFinalVariableName"/>
+        <!--<module name="ConstantName"/>-->
```

Comment:
Hmmm...I guess I changed those because I didn't have the time to see how should I format the Final variables names. I can have a look this weekend and un-comment those fields.